app/src/cards/PopulationCard.tsx

```diff
@@ -44,34 +62,68 @@ function PopulationCard(props: { fips: Fips }) {
                 Missing data means that we don't know the full story.
               </Alert>
             )}
+            {/* Because the Vega charts are using responsive width based on the window resizing,
+                we manually trigger a resize when the div size changes so vega chart will 
+                render with the right size. This means the vega chart won't appear until the 
+                AnimateHeight is finished expanding */}
             {dataset.length > 0 && (
-              <Grid
-                container
-                className={styles.PopulationCard}
-                justify="space-around"
+              <AnimateHeight
+                duration={500}
+                height={expanded ? "auto" : 70}
+                onAnimationEnd={() => window.dispatchEvent(new Event("resize"))}
               >
-                <Grid item>
-                  <span>Total Population</span>
-                  <span className={styles.TotalPopulationValue}>
-                    {totalPopulationSize}
-                  </span>
+                <Grid
```

Comment:
What do you think about wrapping the population names and making it so that the width of each population box is the same? We should check with Chi but to me it looks a bit odd to have them misaligned when they're on two rows

Reply:
Yeah I agree. I did start this as they are aligned on Chi's mocks, but I wasn't sure how long the final names were going to be so I thought I'd postpone it. I'll add a TODO though!